src/assets/data/testimonial/index.js

```diff
@@ -1,5 +1,5 @@
-import image1 from '../../images/app/testimonial/testimonial-thumb-1.jpg';
-import image2 from '../../images/app/testimonial/testimonial-thumb-2.jpg';
+import image1 from '../../images/app/testimonial/intraserve.png';
+import image2 from '../../images/app/testimonial/hpe.png';

```

Comment:
In general, can we try to use SVGs where possible? I understand that they’re likely isn’t one available for intraserve, but for HPE, You can find one in the current Layer5 site.

Reply:
@leecalcote The one present in layer5 repo is a different type of logo, which has the entire full form of HPE(making it rectangular), while we want a square or circle one(also on Internet SVG is available for the rectangular one). So i chose to go with this one.

Reply:
Ok, I'll find one. Thanks, @Jashpatel1